packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -323,23 +325,26 @@ async function getTemplate(templateName: string): Promise<Template> {
     const templateOptionRemote = "<remote zip file>";
     const options = [...templates, templateOptionRemote];
     // Prompt for template
-    const selection = readlineSync.keyInSelect(
-      options,
-      chalkColour`{whiteBright What template you want to use?}`
-    );
-    if (selection == -1) {
-      process.exit(0);
-    }
-    if (selection === options.indexOf(templateOptionRemote)) {
-      templateName = readlineSync.question(
-        "Please enter an URL pointing to the template zip file you want to use: "
-      );
-      if (templateName == "") {
-        console.log("No URL was given (received empty string). Aborted.");
-        process.exit(1);
-      }
```

Comment:
Was this error handling removed on purpose?

Reply:
Yes and no, I forgot to add the validation function :D